addons/website_event_meet/static/src/js/snippets/options.js

```diff
@@ -4,6 +4,23 @@ import options from 'web_editor.snippets.options';
 
 options.registry.WebsiteEvent.include({
 
+    /**
+     * @override
+     */
+    async start() {
+        const res = await this._super(...arguments);
+        const rpcData = await this._rpc({
```

Comment:
A bit sad that now we do 2 rpc instead of one in the previous version, but it's ok, keep it like this.

Reply:
Just FYI. From 17.0, this code has been refactored a bit.
https://github.com/odoo/odoo/blob/0db3290b7fc6a63cfa62f03bbeade9df1bc7dca2/addons/website_event/static/src/snippets/options.js#L20-L34